experiments/ClimaEarth/run_amip.jl

```diff
@@ -336,7 +338,13 @@ if mode_name == "amip"
     )
 
     ## CO2 concentration from temporally varying file
-    CO2_timevaryinginput = TimeVaryingInput(co2_data, "co2", boundary_space, reference_date = date0)
+    CO2_text = DelimitedFiles.readdlm(co2_data, Float64; comments = true)
+    # The text file only has month and year, so we set the day to 15th of the month
+    CO2_dates = Dates.DateTime.(CO2_text[:, 1], CO2_text[:, 2]) + Dates.Day(14)
```

Comment:
For future users/developers, it would be helpful to somehow show which entry of `CO2_text` is the month and which is the year. You could do something like:
```julia
year = CO2_text[:, 1]
month = CO2_text[:, 2]
```
or just include it in a comment

Reply:
It would also be good to include this information about the format of the data in the documentation for the artifact in ClimaArtifacts, if it isn't already there. Including that the CO2 data itself is in the 4th column